stellargraph/__init__.py

```diff
@@ -69,3 +71,41 @@
     "PPNPPropagationLayer": layer.ppnp.PPNPPropagationLayer,
     "APPNPPropagationLayer": layer.appnp.APPNPPropagationLayer,
 }
+
+
+def _top_level_deprecation_warning(name, path):
+    msg = (
+        f"'{name}' is no longer available at the top-level. "
+        f"Please use 'stellargraph.{path}.{name}' instead."
+    )
+    warnings.warn(msg, DeprecationWarning)
```

Comment:
The problem here isn't just caused by the default behaviour of `DeprecationWarning`, but also by point 1 of #1107: setting the `stacklevel` argument to point to the problematic call from the user. If the user calls `stellargraph.expected_calibration_error(...)`, the stack looks like:

1. user code
2. `stellargraph.expected_calibration_error`
3. `_top_level_deprecation_warning`

and so we need 

```suggestion
    warnings.warn(msg, DeprecationWarning, stacklevel=3)
```
to point at the user code.

If I make this change locally, I see the warning locally, and in a Python 3.7 shell.

Reply:
makes sense, okay I'll add stacklevel=3 here